cmd/generate_cmd.go

```diff
@@ -80,36 +102,26 @@ rbac-tool gen --generated-type=ClusterRole --deny-resources=secrets., --allowed-
 	flags.StringSliceVar(&allowedGroups, "allowed-groups", []string{"*"}, "Comma separated list of API groups we would like to allow '*'")
 	flags.StringSliceVar(&allowedVerb, "allowed-verbs", []string{"*"}, "Comma separated list of verbs to include. To include all use '*'")
 	flags.StringSliceVar(&denyResources, "deny-resources", []string{""}, "Comma separated list of resource.group - for example secret. to deny secret (core group) access")
+	flags.StringVarP(&namespace, "namespace", "n", "myNamespace", "Namespace to deploy Role to")
+	flags.StringVarP(&roleName, "roleName", "r", "myRole", "Name of Role or ClusterRole")
+	flags.BoolVarP(&useSubresources, "useSubresources", "s", false, "Include Kubernetes subresources in generated (Cluster)Roles")
 
 	return cmd
 }
 
-func generateRole(generateKind string, rules []rbacv1.PolicyRule) (string, error) {
+func generateRole(rules []rbacv1.PolicyRule, namespace string, roleName string) (string, error) {
 	var obj runtime.Object
 
-	if generateKind == "ClusterRole" {
```

Comment:
This PR removes support for generating cluster roles - is this intended? I'd rather add have this PR extend the existing behavior